app/tests/integration/nimbus/pages/experimenter/audience.py

```diff
@@ -39,6 +44,41 @@ def min_version(self, version=80):
         select = Select(el)
         select.select_by_value(f"FIREFOX_{version}")
 
+    @property
+    def locale(self):
```

Comment:
The instructions in the ticket say:

"So ideally we'd have a test for each targeting config that ... Creates a Nimbus experiment for Desktop with that targeting config and no other custom targeting (we might be able to do this from the command line without needing to start the dev server at all)"

So I don't think we need to touch locales/countries here at all since that falls under 'other custom targeting'.  The only thing we're interested in validating here is the JEXL body of the TargetingConfig entries in NimbusConstants.